src/modules/competition-age-categories/forms/utils.ts

```diff
@@ -0,0 +1,46 @@
+import map from 'just-map-values'
+import { TFunction } from 'next-i18next'
+import * as yup from 'yup'
+
+import { CreateTeamDto } from '@/modules/teams/types'
+
+import {
+  CompetitionAgeCategortyDto,
+  UpdateCompetitionAgeCategoryDto,
+} from '../types'
+
+export const initialValues: CreateTeamDto = {
+  name: '',
+  clubId: 0,
+  competitionId: 0,
+  groupId: 0,
+  minut90url: '',
+  transfermarktUrl: '',
+  lnpId: '',
+}
+
+export function generateCreateValidationSchema(t: TFunction) {
+  return yup
+    .object({
+      name: yup.string().required(t('teams:NO_NAME_ERROR')),
```

Comment:
```suggestion
      name: yup.string().required(t('comp-age-categ:NO_NAME_ERROR')),
```

There's wrong i18n namespace used here and translations don't work properly

![Screenshot 2022-08-09 at 20 40 30](https://user-images.githubusercontent.com/55458549/183738659-612f64a4-d74b-4b7a-bcd7-8f447c2fc171.png)

Please adjust:)

Reply:
added separatly with other changes and accepting suggestion and merge is blocked:c